tests/unit/tests_high_five_parallel.cpp

```diff
@@ -26,16 +26,15 @@
 using namespace HighFive;
 
 
-int argc = boost::unit_test::framework::master_test_suite().argc;
-char** argv = boost::unit_test::framework::master_test_suite().argv;
+static int argc = boost::unit_test::framework::master_test_suite().argc;
+static char** argv = boost::unit_test::framework::master_test_suite().argv;
 
 struct MpiFixture {
     MpiFixture() { MPI_Init(&argc, &argv); }
     ~MpiFixture() { MPI_Finalize(); }
 };
 
-BOOST_GLOBAL_FIXTURE(MpiFixture)
-
+BOOST_GLOBAL_FIXTURE(MpiFixture);
```

Comment:
No,
see https://github.com/BlueBrain/HighFive/pull/254

Reply:
Thanks. It was a local modification that managed to infiltrate the last patch :)